### ISSUE TITLE
fileinfo produces output that cannot be decoded with UTF-8

### ISSUE THREAD

**Issue by s3rvac** [avast-tl/retdec#82]
When `fileinfo` is run over the attached 64-bit ELF in the verbose mode, it produces output that cannot be decoded with UTF-8.

## Input

```sh
$ fileinfo -v FILE > output.txt
```
where `FILE` is [1F9AEA9C8C3A952C86E70BB1BB086AD3FA763F6231C5CDCBA887681403030582](https://www.virustotal.com/intelligence/search/?query=1F9AEA9C8C3A952C86E70BB1BB086AD3FA763F6231C5CDCBA887681403030582).

## Output

When I try to decode the produced output with UTF-8, the decoding fails as there is an invalid character:

```sh
$ python3.6
>>> open('output.txt', 'r', encoding='utf-8').read()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x92 in position 37539: invalid start byte
```
The invalid character is the last non-whitespace character on the following line (below, it is decoded with CP1250):
```
16   String table offset of name of needed library (DT_NEEDED)   0x09b0    libm.s’
```
It's ordinal value is 146 (0x92 hexa).

## Expected output

`fileinfo` produces output that can be decoded with UTF-8. Currently, it is supposed to always produce ASCII output, which is decodable with UTF-8.

## Configuration

* Commit: 8c4b23dbbf0f2baaf6f2309fc189b054f58250ea (current `master`)
* 64b Arch Linux, GCC 7.2.1, Debug build of RetDec

**Comment by s3rvac** [comment]
Another file: [C6F0EFA92E641EA31C08B1F0AA859B7402876B4FA2E65B61E657927F9DC9B6C7](https://www.virustotal.com/intelligence/search/?query=C6F0EFA92E641EA31C08B1F0AA859B7402876B4FA2E65B61E657927F9DC9B6C7)

![c6f0efa92e641ea31c08b1f0aa859b7402876b4fa2e65b61e657927f9dc9b6c7](https://user-images.githubusercontent.com/5675953/35497764-cc7e599c-04cb-11e8-9448-5f62fd0f1319.png)

**Comment by mbandzi** [comment]
Non-printable chars are now replaced with hexadecimal codes as in other parts of retdec-fileinfo.
Fixed in [5eab1d53d7](https://github.com/avast-tl/retdec/commit/5eab1d53d70c76d296ea8e79c980d5f4ba21a7db).
